README.md

```diff
@@ -60,7 +60,18 @@ ENV["MYSQL_DATABASE"] = "test_db"
 
 ## Development
 
-After cloning run `crystal deps` or `crystal deps update`.
+After cloning makes scripts executable:
+
+```
+chmod a+x bin/*
+chmod a+x script/setup-test-db.sh
```

Comment:
I only copied those from the avtive_record.cr readme wasn't entirely sure
why there were in those instructions.

On Tuesday, July 26, 2016, Oleksii Fedorov <notifications@github.com> wrote:

> In README.md
> <https://github.com/waterlink/mysql_adapter.cr/pull/5#discussion_r72383862>
> :
>
> > @@ -60,7 +60,18 @@ ENV["MYSQL_DATABASE"] = "test_db"
> >
> >  ## Development
> >
> > -After cloning run `crystal deps` or `crystal deps update`.
> > +After cloning makes scripts executable:
> > +
> > +```
> > +chmod a+x bin/*
> > +chmod a+x script/setup-test-db.sh
>
> What is the reason for these chmods?
>
> From my knowledge git stores executable flag as part of commit. So if
> somebody checks out the file, that was already made executable before
> checking it in, it should just work. At least, for Linux and MacOS.
>
> Is there some context that I am missing? Some shell, git version, git
> config or OS that I am not aware of working differently in the regard of
> executable flags?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/waterlink/mysql_adapter.cr/pull/5/files/63164dc2ff71d5c60da5004dbd3b5f492fa900e1#r72383862>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAo3I4aeoBwqpBx0h4mJhdejmJ5IaZ7ks5qZvGvgaJpZM4JVsWT>
> .
>